Services/Administration/OJS.Services.Administration.Business/IOrderableService.cs

```diff
@@ -0,0 +1,10 @@
+namespace OJS.Services.Administration.Business;
+
+using SoftUni.Services.Infrastructure;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+public interface IOrderableService<TEntity> : IService
```

Comment:
Why make a separate service for ordering instead of making these functions private methods of the already existing business service?

Reply:
I think the whole point was to have an additional service for future orderings. @gogo4ds, please give your opinion on how I should proceed.

Reply:
The logic for ordering by `OrderBy` is not model specific and should be the same for all entities. That's why is better to have single reusable service for doing it.

Reply:
The logic for ordering by `OrderBy` is not model specific and should be the same for all entities. That's why is better to have single reusable service for doing it.